api/app/tests/hfi/test_hfi_endpoint_request.feature

```diff
@@ -7,8 +7,11 @@ Feature: /hfi/
         And the response is <response_json>
 
         Examples:
-            | request_json                   | status_code | response_json                   |
+            # TODO: These test currently exposes a "bug" in the code where removing a station from one area, means it's removed from all.
```

Comment:
Do you mean when a station is deselected in one fire centre it gets deselected in all other fire centres that use that same station?

Reply:
No, planning area, not fire centre.